tap_github/streams.py

```diff
@@ -270,17 +270,33 @@ class CommunityProfileStream(GitHubStream):
 class LanguagesStream(GitHubStream):
     name = "languages"
     path = "/repos/{org}/{repo}/languages"
-    primary_keys = ["repo", "org"]
+    primary_keys = ["repo", "org", "language_name"]
     parent_stream_type = RepositoryStream
     ignore_parent_replication_key = False
     state_partitioning_keys = ["repo", "org"]
 
+    def parse_response(self, response: requests.Response) -> Iterable[dict]:
+        """Parse the language response and reformat to return as an iterator of [{language_name: Python, byes: 23}]."""
+        if response.status_code in self.tolerated_http_errors:
+            return []
+
+        languages_json = response.json()
+        results = [
+            {"language_name": key, "bytes": value}
+            for key, value in languages_json.items()
+        ]
+
+        for row in results:
+            yield row
```

Comment:
```suggestion
        for key, value in languages_json.items()
            yield {"language_name": key, "bytes": value}
```
Just to save a bit of memory :)